plugin/trino-raptor-legacy/src/test/java/io/trino/plugin/raptor/legacy/RaptorQueryRunner.java

```diff
@@ -64,15 +64,14 @@ public static QueryRunner createRaptorQueryRunner(
         queryRunner.createCatalog("tpch", "tpch");
 
         queryRunner.installPlugin(new RaptorPlugin());
-        File baseDir = queryRunner.getCoordinator().getBaseDataDir().toFile();
         Map<String, String> raptorProperties = ImmutableMap.<String, String>builder()
                 .putAll(extraRaptorProperties)
                 .put("metadata.db.type", "h2")
-                .put("metadata.db.filename", new File(baseDir, "db").getAbsolutePath())
-                .put("storage.data-directory", new File(baseDir, "data").getAbsolutePath())
+                .put("metadata.db.filename", Files.createTempDirectory("raptor-db").toAbsolutePath().toString())
+                .put("storage.data-directory", Files.createTempDirectory("raptor-data").toAbsolutePath().toString())
                 .put("storage.max-shard-rows", "2000")
                 .put("backup.provider", "file")
-                .put("backup.directory", new File(baseDir, "backup").getAbsolutePath())
+                .put("backup.directory", Files.createTempDirectory("raptor-backup").toAbsolutePath().toString())
```

Comment:
Do we need to delete these temp directories?

(you can probably use runner's onClose to do cleanup)